addons/bsp_importer/bsp_reader.gd

```diff
@@ -2109,8 +2110,10 @@ func get_verts(vert_bytes : PackedByteArray) -> PackedVector3Array:
 			var xbytes = bytes(vert_bytes, range( (v * 12) + 0, (v * 12) + 4 )).decode_float(0)
 			var ybytes = bytes(vert_bytes, range( (v * 12) + 4, (v * 12) + 8 )).decode_float(0)
 			var zbytes = bytes(vert_bytes, range( (v * 12) + 8, (v * 12) + 12 )).decode_float(0)
+			#TODO it should work with _unit_scale and not with inverse_scale_fac*_unit_scale.
+			# i guess someone did some multiplacations elsewhere? i couldn't trace them, this must be fixed
+			var vec = convert_vector_from_quake_scaled(Vector3(xbytes, ybytes, zbytes), inverse_scale_fac*_unit_scale)
```

Comment:
This is the same as using 1.  What is this vector used for?  Looks like the original code did not scale it, either.  Should probably just used the unscaled function.